File: DATOS ABSTRACTOS/DATOS ABS.py
class Persona:
    def __init__(self, nombre, edad):
        self.nombre = nombre
        self.edad = edad
   
    def saludar(self):
        return f"Hola, soy {self.nombre} y tengo {self.edad} años."


# Uso del TDA
p1 = Persona("Gustavo", 20)
print(p1.saludar())

File: DATOS ABSTRACTOS/DATOS ABS.js
class Persona {
    constructor(nombre, edad) {
        this.nombre = nombre;
        this.edad = edad;
    }
    saludar() {
        return `Hola, soy ${this.nombre} y tengo ${this.edad} años.`;
    }
}


// Uso del TDA
let p1 = new Persona("Gustavo", 20);
console.log(p1.saludar());
